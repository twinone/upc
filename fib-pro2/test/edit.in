+ 30.05.20 10:30 hi
+ 31.05.20 00:00 ciao
?
1 01.06.20 00:05 new name
1 #important
?
rellotge 01.06.20 00:06
1 #bla
sortir

##############
Author Twinone

Tests:
If we query and change the time of an element so that it would come up
in a different position in the next query, we must still be able to 
refer to that task by the same id (test if menu is not a set)

Tests whether you can modify a task that was added to the menu
when it was in the future but is now part of the past

Tests whether you can edit a task's clock to the past

Test whether you can edit a task's clock to an existing task
