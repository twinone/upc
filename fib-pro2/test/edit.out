
+ 30.05.20 10:30 hi

+ 31.05.20 00:00 ciao

?
1 hi 30.05.20 10:30
2 ciao 31.05.20 00:00

1 01.06.20 00:05 new name

1 #important

?
1 ciao 31.05.20 00:00
2 new name 01.06.20 00:05 #important

rellotge 01.06.20 00:06

1 #bla
No s'ha realitzat

sortir
